### PR TITLE
Getting started gcr

### PR DESCRIPTION
**Pull request by goodbyedavid**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: docs/provision-in-gcr.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    - Amazon QuickSight is not availible in GCR regions.
+
+## Provision AWS SaaS Boost into your AWS GCR Account
+To prepare the installation process, perform the following steps as needed:
+1. You have a domain and your domain is [ICP registrated](https://www.amazonaws.cn/en/about-aws/china/#ICP_).
+2. You need a root domain name and individual domain names for your keycloak installation and the admin web UI. As part of this you need one certificate in ACM for keycloak domain, and one certificate in IAM that covers admin web UI domain. Either wildcard certificates (e.g. *.example.com) or specific certificates (e.g. keycloak.example.com). 
+    - You need to create a public hosted zone in AWS Route53 for your domain.
+    - Request or upload [certificate to ACM](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html)
+    - Upload [certificate to IAM](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-procedures.html). 
+    - If you don't have a certificate availible, [letsencrypt.org](https://letsencrypt.org/) provides free TLS certificates.
```
Comment (PoeppingT) [inline docs/provision-in-gcr.md:34]: We need to be careful about explicitly endorsing a particular TLS certificate provider here. Instead, let's say `There are many free and paid options you can use to generate a TLS certificate. The example we use here is [letsencrypt.org](https://letsencrypt.org/).`

File: docs/getting-started.cn.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
+# 亚马逊云科技 SaaS Boost 入门指南
```
Comment (PoeppingT) [inline docs/getting-started.cn.md:1]: Please get a second reviewer for this piece to double check your work.

**Review by PoeppingT — CHANGES_REQUESTED** [review]
Overall LGTM, just a few required changes and we can merge.

File: docs/provision-in-gcr.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    - Amazon QuickSight is not availible in GCR regions.
+
+## Provision AWS SaaS Boost into your AWS GCR Account
+To prepare the installation process, perform the following steps as needed:
+1. You have a domain and your domain is [ICP registered](https://www.amazonaws.cn/en/about-aws/china/#ICP_in_China).
+2. You need a root domain name and individual domain names for your keycloak installation and the admin web UI. As part of this you need one certificate in ACM for keycloak domain, and one certificate in IAM that covers admin web UI domain. Either wildcard certificates (e.g. *.example.com) or specific certificates (e.g. keycloak.example.com). 
+    - You need to create a public hosted zone in AWS Route53 for your domain.
+    - Request or upload [certificate to ACM](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html)
+    - Upload [certificate to IAM](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-procedures.html). 
+    - There are many free and paid options you can use to generate a SSL/TLS certificate. The example we use here is [letsencrypt.org](https://letsencrypt.org/) for a free SSL/TLS certificate.
```
Comment (PoeppingT) [inline docs/provision-in-gcr.md:33]: Actually @goodbyedavid after some more conversation in the team we've decided it's best if we leave this out completely. It's safer for the project from a liability perspective if we allow customers to discover their own mechanism for generating SSL/TLS certificates. Please remove this, and we can merge this PR.

Comment (goodbyedavid) [reply docs/provision-in-gcr.md]: @PoeppingT This part is now removed.

**Review by PoeppingT — APPROVED** [review]
LGTM, thanks for doing this!

**Pull request merged**